lib/iris/tests/unit/lazy_data/test_co_realise_cubes.py

```diff
@@ -71,15 +72,29 @@ def test_multi(self):
         self.assertTrue(cube_inner.has_lazy_data())
 
     def test_combined_access(self):
+        import dask
+        from distutils.version import StrictVersion as Version
+
         wrapped_array = ArrayAccessCounter(np.arange(3.))
         lazy_array = as_lazy_data(wrapped_array)
         derived_a = lazy_array + 1
         derived_b = lazy_array + 2
+        derived_c = lazy_array + 3
         cube_a = Cube(derived_a)
         cube_b = Cube(derived_b)
-        co_realise_cubes(cube_a, cube_b)
-        # Though used twice, the source data should only get fetched once.
-        self.assertEqual(wrapped_array.access_count, 1)
+        cube_c = Cube(derived_c)
+        co_realise_cubes(cube_a, cube_b, cube_c)
+        # Though used more than once, the source data should only get fetched
+        # once by dask.
+        if Version(dask.__version__) < Version('2.0.0'):
+            self.assertEqual(wrapped_array.access_count, 1)
```

Comment:
Do we really need this older form?
Instead we could just specify dask>=2 ??

Reply:
@pp-mo `dask` version `2.+` is Python3 only. 

As you know, in Travis CI we currently test Python `2.7`, `3.6` and now `3.7` - so yes, we do need this to cover the Travis CI testing split between Python2 and Python3... for now.

I've created the issue https://github.com/SciTools/iris/issues/3356 to tidy this for `iris` version `3.0.0`.

Reply:
Ok, understood
Shame though!